src/components/SearchBox.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { connectSearchBox } from 'react-instantsearch/connectors';
+import * as colors from 'kitsu/constants/colors';
+import { Item, Input, Icon } from 'native-base';
+
+export default connectSearchBox(({ refine, currentRefinement, placeholder }) => (
+  <Item style={styles.searchBoxContainer}>
+    <Icon name="ios-search" style={styles.searchBoxIcon} />
+    <Input
+      placeholder={placeholder}
+      value={currentRefinement}
+      onChangeText={t => refine(t)}
+      style={styles.searchBoxInput}
+      autoCapitalize={'none'}
+      autoCorrect={false}
+      underlineColorAndroid={'transparent'}
+      placeholderTextColor={colors.placeholderGrey}
+    />
+  </Item>
+));
+
+const styles = {
```

Comment:
You can call flatten on the style and send it into native base components. 

Fwiw: I would try to avoid using native base all together if you can.